applets/002-save-sysctls/applet.py

```diff
@@ -1,21 +1,22 @@
 
 def main(x, conf, args):
     if not len(args) > 0:
-        x.p.msg("no hostname argument given, like alpha\n")
+        x.p.err("no hostname argument given, like alpha\n")
         return False
 
     hostname = args[0]
     ip = conf.get_control_ip(hostname)
     user = conf.get_user(hostname)
     # check if file exist already, if we break here
     # the following four lines are probably obsolete
-    _, _, exit_code = x.ssh.exec(ip, user, "test -f /tmp/sysctl.save")
+    exit_code = x.ssh.exec(ip, user, "test -f /tmp/sysctl.save")
```

Comment:
is that correct? exit_code holds a tupple now and is always correct!

The following should work

```
exit_code = x.ssh.exec(ip, user, "test -f /tmp/sysctl.save")[2]
```

or even better readable:

```
ret = x.ssh.exec(ip, user, "test -f /tmp/sysctl.save")
if ret[2] == 0
```